bundles/XliffBundle/src/Controller/XliffTranslationController.php

```diff
@@ -0,0 +1,170 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\XliffBundle\Controller;
+
+use Pimcore\Bundle\AdminBundle\Controller\Admin\TranslationController;
+use Pimcore\Logger;
+use Pimcore\Model\Element;
+use Pimcore\Bundle\XliffBundle\ExportService\Exporter\ExporterInterface;
+use Pimcore\Bundle\XliffBundle\ExportService\ExportServiceInterface;
+use Pimcore\Bundle\XliffBundle\ImportDataExtractor\ImportDataExtractorInterface;
+use Pimcore\Bundle\XliffBundle\ImporterService\ImporterServiceInterface;
+use Pimcore\Bundle\XliffBundle\TranslationItemCollection\TranslationItemCollection;
+use Symfony\Component\HttpFoundation\BinaryFileResponse;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\ResponseHeaderBag;
+use Symfony\Component\Routing\Annotation\Route;
+
+class XliffTranslationController extends TranslationController
+{
+    /**
+     * @Route("/xliff-export", name="pimcore_bundle_xliff_translation_xliffexport", methods={"POST"})
+     *
+     * @param Request $request
+     * @param ExportServiceInterface $exportService
+     *
+     * @return JsonResponse
+     *
+     * @throws \Exception
+     */
+    public function xliffExportAction(Request $request, ExportServiceInterface $exportService): JsonResponse
+    {
+        $this->checkPermission('xliff_import_export');
+        
+        $id = $request->get('id');
+        $data = $this->decodeJson($request->get('data'));
+        $source = $request->get('source');
+        $target = $request->get('target');
+
+        $translationItems = new TranslationItemCollection();
+
+        foreach ($data as $el) {
+            $element = Element\Service::getElementById($el['type'], $el['id']);
+            $translationItems->addPimcoreElement($element);
+        }
+
+        $exportService->exportTranslationItems($translationItems, $source, [$target], $id);
+
+        return $this->adminJson([
+            'success' => true,
+        ]);
+    }
+
+    /**
+     * @Route("/xliff-export-download", name="pimcore_bundle_xliff_exportdownload", methods={"GET"})
```

Comment:
```suggestion
     * @Route("/xliff-export-download", name="pimcore_bundle_xliff_translation_exportdownload", methods={"GET"})
```
Also, add route prefix `/translation` to the controller class to justify the route names here.